bootstrap/pkg/kfapp/gcp/gcp.go

```diff
@@ -117,6 +118,10 @@ type Setter interface {
 	SetRunGetCredentials(v bool)
 }
 
+func (gcp *Gcp) GetKfConfig() *kfconfig.KfConfig {
```

Comment:
Why the new method?

Reply:
So outside can access kfconfig. any conern?